packages/core/src/base-axis-chart.ts

```diff
@@ -62,6 +66,8 @@ export class BaseAxisChart extends BaseChart {
 			this.drawXGrid();
 			this.drawYGrid();
 
+			this.createSlider();
```

Comment:
Function names should be more readable and understandable. Also, the function should only be called if a certain option exists in the chart configurations. e.g.:
```js
if (this.options.scales.zoom) {
  this.makeChartZoomable();
}
```